<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>524</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backlink machine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="input_keyword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#input_keyword{border:2px groove red}
#input_keyword{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="placeholderText">
     <string>এখানে মেইন কিওয়ার্ড দিন</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(255, 0, 0);
border-radius:5px 5px 5px 5px;
font-size:13px;</string>
    </property>
    <property name="text">
     <string>Find Guestpost Opportunity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_mass_email_open_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(255, 0, 0);
border-radius:5px 5px 5px 5px;
font-size:13px;</string>
    </property>
    <property name="text">
     <string>Send Mass Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_email_account_window_opener_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(255, 0, 0);
border-radius:5px 5px 5px 5px;
font-size:13px;</string>
    </property>
    <property name="text">
     <string>Add Email Account</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font-size:47px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;BACKLINK MACHINE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_quant">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#input_quant{border:2px groove red}
#input_quant{background-color:rgb(255, 255, 255)}</string>
    </property>
    <property name="placeholderText">
     <string>কয়টা সাইট লাগবে?</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
